tmt/steps/provision/__init__.py

```diff
@@ -885,6 +885,9 @@ def setup(self) -> None:
 
         Setup the guest after it has been started. It is called after :py:meth:`Guest.start`.
         """
+        pid_directory = '/var/tmp/pid'  # noqa: S108 insecure usage of temporary dir
```

Comment:
This line shouldn't exist, it duplicates the already defined `TEST_PIDFILE_ROOT` constant and `effective_pidfile_root()` function that includes possible user change. Both commands below should use `effective_pidfile_root()` instead.

Reply:
addressed